geometry_msgs/Vector3 DirectionVector
geometry_msgs/Point CentorPoint
